### PR TITLE
Remove type hints for `\Illuminate\Support\Collection`

### PR DESCRIPTION
**Pull request by strategio**
This will prevent fatal errors if a new version of
`tightenco/collect` is already loaded.

https://onthegosystems.myjetbrains.com/youtrack/issue/wpmlcore-6677

### CODE REVIEW

**Comment by strategio** [comment]
@OnTheGoSystems/wpml please review.

File: src/reusable-blocks/class-job-links.php
Comment (andreasciamanna) [inline src/reusable-blocks/class-job-links.php:20]: @strategio

I think the problem is the `\` before `collect`.

The `use Illuminate\Support\Collection` should be enough to avoid any ambiguity, but if is not, why you can't just use `\Illuminate\Support\Collection\collect( $job_ids )->map( function( $job_id )` here and wherever you use this function?

**Review by andreasciamanna — CHANGES_REQUESTED** [review]

File: src/reusable-blocks/class-job-links.php
Comment (andreasciamanna) [reply src/reusable-blocks/class-job-links.php:20]: Now I understand that `collect` is actually declared only in the global namespace, not with a specific one.

I still don't like this fix, but I can't even think of something better, I'm afraid.

**Review by andreasciamanna — APPROVED** [review]
If there isn't really a better way...

**Comment by andreasciamanna** [comment]
Please wait for Bruce's feedback before merging

File: src/reusable-blocks/class-job-links.php
Comment (strategio) [reply src/reusable-blocks/class-job-links.php:20]: Yes @andreasciamanna, `collect` is in the global namespace.

See in the latest version https://github.com/tightenco/collect/blob/laravel-5.8-ongoing/src/Collect/Support/helpers.php#L21, it returns `\Tightenco\Collect\Support\Collection`. In the version we have in Core, it returns `\Illuminate\Support\Collection` (I can't find it on the GitHub repo).

**Comment by strategio** [comment]
Closing here, was handled in https://github.com/OnTheGoSystems/wpml-page-builders-gutenberg/pull/53.

**Pull request closed without merging**
